code/datums/mutations/telekinesis.dm

```diff
@@ -11,8 +11,8 @@
 /datum/mutation/human/telekinesis/New(class_ = MUT_OTHER, timer, datum/mutation/human/copymut)
 	..()
 	if(!(type in visual_indicators))
-		visual_indicators[type] = list(mutable_appearance('icons/effects/genetics.dmi', "telekinesishead", -MUTATIONS_LAYER))
-
+		visual_indicators[type] = list(mutable_appearance('modular_skyrat/master_files/icons/effects/tele_effects.dmi', "telekinesishead", -MUTATIONS_LAYER)) //SKYRAT EDIT, Old icon location: "icons/effects/genetics.dmi"
```

Comment:
:-1:
Get a better sprite for this that's very visible. The current one is too hard to see.

Reply:
Okay, I saw the new sprite in the preview image, but make it pulse way more often and never fade out completely.